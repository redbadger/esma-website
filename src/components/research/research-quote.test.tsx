import React from "react";
import renderer from "react-test-renderer";
import PrevNext from "./prev-next";
import ResearchQuote from './research-quote';

describe("ResearchQuote", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <ResearchQuote source="Sample source">
            <p>Sample quote</p>
        </ResearchQuote>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
