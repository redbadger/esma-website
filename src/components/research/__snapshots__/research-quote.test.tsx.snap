// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResearchQuote renders correctly 1`] = `
<div
  className="css-soql84-containerStyles"
>
  <div
    className="css-1fjfpf6-researchQuoteStyles"
  >
    <span
      className="opening-quote quote-mark"
    />
    <div
      className="quote-content"
    >
      <p>
        Sample quote
      </p>
      <span
        className="closing-quote quote-mark"
      />
    </div>
  </div>
  <p
    className="quote-source"
  >
    Sample source
  </p>
</div>
`;
